buildSrc/src/main/resources/META-INF/schemas/guide-metadata.schema.json

```
@@ -0,0 +1,206 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "uri": "https://guides.micronaut.io/schemas",
+  "type": "object",
+  "required": ["title","intro","authors","categories","publicationDate"],
+  "properties": {
+    "title": {
+      "description": "The guide's title",
+      "type": "string"
+    },
+    "authors": {
+      "description": "The guide's authors",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "intro": {
+      "description": "The guide introduction",
+      "type": "string"
+    },
+    "features": {
+      "description": "The features that the guide requires",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "publicationDate": {
+      "description": "The guide publication date. It should follow the format YYYY-MM-DD",
+      "type": "string",
+      "format": "date",
+      "pattern": "^\\d{4}\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
+    },
+    "minimumJavaVersion": {
+      "description": "If the guide needs a minimum Java version, define it here.",
+      "type": "integer"
+    },
+    "maximumJavaVersion": {
+      "description": "If the guide needs a maximum Java version, define it here.",
+      "type": "integer"
+    },
+    "prerequisites": {
+      "description": "The prerequisites of the guide",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
```

delete this for now: 

```suggestion
    "prerequisites": {
      "description": "The prerequisites of the guide",
      "type": "array",
      "items": {
        "type": "string"
      }
    },
```